operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -191,12 +191,14 @@ spec:
 #      url: ""
 # **Tracing-specific settings:
 #  - Right now we only support Jaeger
+# enabled: When true, Kiali shows Jaeger and autodiscover
```

Comment:
Trivial - but can we reword this slightly? The sentence seems to be cut off here. Say something like:

```suggestion
# enabled: When true, Kiali shows Jaeger and will attempt to autodiscover it.
```